src/AzurePipelinesToGitHubActionsConverter.Core/GitHubActions/Step.cs

```diff
@@ -19,11 +17,14 @@ public string run
             }
             set
             {
-                //Spaces on the beginning or end seem to be a problem for the YAML serialization
                 if (string.IsNullOrEmpty(value) == false)
                 {
-                    value = value.Trim();
+                    //Spaces on the beginning or end seem to be a problem for the YAML serialization, so we Trim() here
```

Comment:
@NateRickard small nit - please make spacing between // and text consistently 1 space

Reply:
that's how I format most of my comments (though not this one), but the older code is mostly the other way fwiw

Reply:
I updated comments that were mine, and some others that were nearby... I suggest we open another PR if we want to equalize throughout the remainder of the codebase.